src/prefect/tasks/prefect/flow_run_cancel.py

```diff
@@ -0,0 +1,38 @@
+from prefect import Task
+from prefect.client import Client
+from typing import Any
+import prefect
+
+
+class CancelFlowRunTask(Task):
+    """
+    Task to cancel a flow.
```

Comment:
Can you add something to this docstring to mention that if a flow run id is not passed in then `flow_run_id` will be pulled from context which effectively allows you to cancel the current flow run this is running in